packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -99,24 +98,22 @@ const _Dropdown = ({
 
   const dropdownBaseId = useId('dropdown');
 
-  useDidUpdate(() => {
-    onOpenChange?.(isOpen);
-    if (!isOpen) {
+  const setIsOpenControlled = (isControlledStateOpen: boolean): void => {
+    onOpenChange?.(isControlledStateOpen);
+    if (!isControlledStateOpen) {
```

Comment:
Can we use the useControllableState hook to manage the controlled/uncontrolled behaviour of `isOpen`?

Reply:
ouu didn't know about this hook. let me see.

Reply:
done